experimental/oak_functions_with_envoy/client/client.yaml

```diff
@@ -0,0 +1,47 @@
+static_resources:
+  listeners:
+    # Accepts TCP streams and tunnels them over HTTP streams.
```

Comment:
"tunnels each of them over ...." is more precise, since it says what hapens to each of them as opposed to all of them in aggregate